<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyRPG Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/share/old_easyrpg.png</normaloff>:/icons/share/old_easyrpg.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game &amp;Project</string>
    </property>
    <addaction name="action_New_Project"/>
    <addaction name="action_Open_Project"/>
    <addaction name="action_Close_Project"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Game_Disk"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuMaps">
    <property name="title">
     <string>Maps</string>
    </property>
    <widget class="QMenu" name="menuEdition_Mode">
     <property name="title">
      <string>Edition &amp;Mode</string>
     </property>
     <addaction name="action_Lower_Layer"/>
     <addaction name="action_Upper_Layer"/>
     <addaction name="action_Events"/>
    </widget>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionScale_1_1"/>
     <addaction name="actionScale_1_2"/>
     <addaction name="actionScale_1_4"/>
     <addaction name="actionScale_1_8"/>
    </widget>
    <addaction name="action_Save_all_Maps"/>
    <addaction name="actionRevert_all_Maps"/>
    <addaction name="separator"/>
    <addaction name="menuEdition_Mode"/>
    <addaction name="menuZoom"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionData_Base"/>
    <addaction name="actionResource_Manager"/>
    <addaction name="actionJukebox"/>
    <addaction name="actionSearch"/>
    <addaction name="action_Script_Editor"/>
   </widget>
   <widget class="QMenu" name="menuGame_2">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="action_Play_Test"/>
    <addaction name="separator"/>
    <addaction name="action_Full_Screen"/>
    <addaction name="action_Title_Background"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPalete"/>
    <addaction name="actionMap_Tree"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionChipset"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuMaps"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuGame_2"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New_Project"/>
   <addaction name="action_Open_Project"/>
   <addaction name="action_Close_Project"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_Game_Disk"/>
   <addaction name="separator"/>
   <addaction name="action_Save_all_Maps"/>
   <addaction name="actionRevert_all_Maps"/>
   <addaction name="separator"/>
   <addaction name="action_Lower_Layer"/>
   <addaction name="action_Upper_Layer"/>
   <addaction name="action_Events"/>
   <addaction name="separator"/>
   <addaction name="actionScale_1_1"/>
   <addaction name="actionScale_1_2"/>
   <addaction name="actionScale_1_4"/>
   <addaction name="actionScale_1_8"/>
   <addaction name="separator"/>
   <addaction name="actionData_Base"/>
   <addaction name="action_Script_Editor"/>
   <addaction name="actionResource_Manager"/>
   <addaction name="actionJukebox"/>
   <addaction name="actionSearch"/>
   <addaction name="separator"/>
   <addaction name="action_Play_Test"/>
   <addaction name="separator"/>
   <addaction name="action_Full_Screen"/>
   <addaction name="action_Title_Background"/>
   <addaction name="separator"/>
   <addaction name="actionContents"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockPalete">
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Palette</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockMapTree">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Map Tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeMap">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Map ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Map Name</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>0</string>
        </property>
        <property name="text">
         <string>Root</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionZoom"/>
   <addaction name="actionDraw"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="actionFill"/>
  </widget>
  <action name="action_Open_Project">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_open.png</normaloff>:/icons/share/old_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="statusTip">
    <string>Open an existing Game Project</string>
   </property>
  </action>
  <action name="action_New_Project">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_new.png</normaloff>:/icons/share/old_new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="statusTip">
    <string>Create a new Game Project</string>
   </property>
  </action>
  <action name="action_Close_Project">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_close.png</normaloff>:/icons/share/old_close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
   <property name="statusTip">
    <string>Close current opened Game Project</string>
   </property>
  </action>
  <action name="actionCreate_Game_Disk">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_create_game_disk.png</normaloff>:/icons/share/old_create_game_disk.png</iconset>
   </property>
   <property name="text">
    <string>Create Game &amp;Disk</string>
   </property>
   <property name="statusTip">
    <string>Generate game installation executable</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_quit.png</normaloff>:/icons/share/old_quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit EasyRPG</string>
   </property>
  </action>
  <action name="action_Save_all_Maps">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_save_all.png</normaloff>:/icons/share/old_save_all.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save all Maps</string>
   </property>
   <property name="statusTip">
    <string>Save all changes done in maps</string>
   </property>
  </action>
  <action name="actionRevert_all_Maps">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_revert.png</normaloff>:/icons/share/old_revert.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Revert all Maps</string>
   </property>
   <property name="statusTip">
    <string>Revert changes on all maps</string>
   </property>
  </action>
  <action name="action_Lower_Layer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_lower_layer.png</normaloff>:/icons/share/old_lower_layer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Lower Layer</string>
   </property>
   <property name="statusTip">
    <string>Edit lower layer mode</string>
   </property>
  </action>
  <action name="action_Upper_Layer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_upper_layer.png</normaloff>:/icons/share/old_upper_layer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Upper Layer</string>
   </property>
   <property name="statusTip">
    <string>Edit upper layer mode</string>
   </property>
  </action>
  <action name="action_Events">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_event_layer.png</normaloff>:/icons/share/old_event_layer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Events</string>
   </property>
   <property name="statusTip">
    <string>Edit events mode</string>
   </property>
  </action>
  <action name="actionScale_1_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_11_scale.png</normaloff>:/icons/share/old_11_scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale 1/&amp;1</string>
   </property>
   <property name="statusTip">
    <string>Scale 200%</string>
   </property>
  </action>
  <action name="actionScale_1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_12_scale.png</normaloff>:/icons/share/old_12_scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale 1/&amp;2</string>
   </property>
   <property name="statusTip">
    <string>Scale 100%</string>
   </property>
  </action>
  <action name="actionScale_1_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_14_scale.png</normaloff>:/icons/share/old_14_scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale 1/&amp;4</string>
   </property>
   <property name="statusTip">
    <string>Scale 50%</string>
   </property>
  </action>
  <action name="actionScale_1_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_18_scale.png</normaloff>:/icons/share/old_18_scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale 1/&amp;8</string>
   </property>
   <property name="statusTip">
    <string>Scale 25%</string>
   </property>
  </action>
  <action name="actionData_Base">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_database.png</normaloff>:/icons/share/old_database.png</iconset>
   </property>
   <property name="text">
    <string>Data Base</string>
   </property>
   <property name="statusTip">
    <string>Open Data Base editor</string>
   </property>
  </action>
  <action name="actionResource_Manager">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_resources.png</normaloff>:/icons/share/old_resources.png</iconset>
   </property>
   <property name="text">
    <string>Resource Manager</string>
   </property>
   <property name="statusTip">
    <string>Open Resource Manager</string>
   </property>
  </action>
  <action name="actionJukebox">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_music.png</normaloff>:/icons/share/old_music.png</iconset>
   </property>
   <property name="text">
    <string>Jukebox</string>
   </property>
   <property name="statusTip">
    <string>Play music while you work</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_search.png</normaloff>:/icons/share/old_search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="statusTip">
    <string>Search inside your Game Project</string>
   </property>
  </action>
  <action name="action_Play_Test">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_playtest.png</normaloff>:/icons/share/old_playtest.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play Test</string>
   </property>
   <property name="statusTip">
    <string>Test your Game</string>
   </property>
  </action>
  <action name="action_Full_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_fullscreen.png</normaloff>:/icons/share/old_fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
   <property name="statusTip">
    <string>Test game in fullscreen mode</string>
   </property>
  </action>
  <action name="action_Title_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_title.png</normaloff>:/icons/share/old_title.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Title Background</string>
   </property>
   <property name="statusTip">
    <string>Show title screen's background on Play Test</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_help.png</normaloff>:/icons/share/old_help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="statusTip">
    <string>View Help contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="statusTip">
    <string>About EasyRPG</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_Script_Editor">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_script_editor.png</normaloff>:/icons/share/old_script_editor.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Script Editor</string>
   </property>
   <property name="statusTip">
    <string>Open Script Editor</string>
   </property>
  </action>
  <action name="actionPalete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_palette.png</normaloff>:/icons/share/old_palette.png</iconset>
   </property>
   <property name="text">
    <string>Palete</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide palette window</string>
   </property>
  </action>
  <action name="actionMap_Tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_map.png</normaloff>:/icons/share/old_map.png</iconset>
   </property>
   <property name="text">
    <string>Map Tree</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide Map Tree window</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_revert.png</normaloff>:/icons/share/old_revert.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last action</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_select.png</normaloff>:/icons/share/old_select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select and move part of the map</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_zoom.png</normaloff>:/icons/share/old_zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom in and out</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_pen.png</normaloff>:/icons/share/old_pen.png</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="toolTip">
    <string>Draw in the map with a pen</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_rectangle.png</normaloff>:/icons/share/old_rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Draw in the map with a rectangle tool</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_circle.png</normaloff>:/icons/share/old_circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>Draw in a map with a circle tool</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_fill.png</normaloff>:/icons/share/old_fill.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="toolTip">
    <string>Draw in the map with a fill tool</string>
   </property>
  </action>
  <action name="actionChipset">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/share/old_palette.png</normaloff>:/icons/share/old_palette.png</iconset>
   </property>
   <property name="text">
    <string>Chipset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
